sequence_transform.c

```diff
@@ -295,8 +411,6 @@ bool process_sequence_transform(uint16_t keycode, keyrecord_t *record, uint16_t
 #if SEQUENCE_TRANSFORM_IDLE_TIMEOUT > 0
     sequence_timer = timer_read32();
 #endif
-    if (!record->event.pressed)
-        return true;
```

Comment:
do we no longer want this general catch here?

Reply:
I needed to modify it to allow handling of backspace released events, and I thought it made more sense for it to live with the other checks in st_process_check